index.css

```diff
@@ -0,0 +1,332 @@
+@import url("https://fonts.googleapis.com/css2?family=Lato:wght@100;300;700;900&display=swap");
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.section {
+  height: 84vh;
+}
+
+body {
+  font-family: "Lato", sans-serif;
+}
+
+nav {
+  display: flex;
+  background-color: #333;
+  justify-content: space-between;
+  font-size: 19px;
+}
+
+.nav-ul {
+  list-style-type: none;
+  margin-right: 4px;
+  padding: 0;
+  overflow: hidden;
+  display: flex;
+}
+
+.title a {
+  color: white;
+  padding: 14px 16px;
+  text-decoration: none;
+}
+
+.nav-ul li a {
+  display: block;
+  color: white;
+  padding: 14px 16px;
+  text-decoration: none;
+}
+
+.nav-ul li {
+  border-right: 1px solid #bbb;
+}
+
+.contact-detail li {
+  font-size: 18px;
+}
+
+.nav-ul li:last-child {
+  border-right: none;
+}
+
+.nav-ul li a:hover {
+  background-color: #111;
+}
+
+.title {
+  margin-left: 15px;
+  display: flex;
+  align-items: center;
+}
+
+.body-container {
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin-top: 50px;
+}
+
+.book-container {
+  border: 1px solid gray;
+  display: flex;
+  flex-direction: column;
+  gap: 10px;
+  width: 50%;
+
+  /* padding: 20px; */
```

Comment:
Great job @betses 🤝

- [ ] You may kindly remove all commented code from your project as this is not regarded as a good practice.